src/styles/blocks/card.scss

```diff
@@ -0,0 +1,59 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 40px 0;
+
+  &__img {
+    height: 280px;
+    width: 280px;
+    border-radius: 4px;
+    margin-bottom: 24px;
+    transition: 0.5s;
+
+  }
+
+  &__description {
+    font-family: Manrope, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 22.4px;
+    letter-spacing: 0.15px;
+    color: #1B2129;
+    margin: 8px 0 12px;
+    transition: 0.2s;
+  }
+
+  &:hover {
+    .card__img {
+      transform: scale(1.05);
+      border-radius: 10px;
+      box-shadow: 4px 4px 10px rgba(106, 85, 67, 0.3);
+    }
+
+    .card__description {
+      color: #362718;
+    }
+  }
+
+  @include laptop {
+    width: 600px;
+    height: 757px;
```

Comment:
no need to define sizes using px, it's not responsive, by default it will fit content